plugins/replicas.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: replicas
```

Comment:
Violates naming guidelines. Others can write other plugins named replicas. Be descriptive.